concourse/tasks/tag-commit.yaml

```diff
@@ -0,0 +1,19 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/gcp-guest/daisy-builder
+    tag: latest
+
+inputs:
+- name: token
+
+
+run:
+  path: sh
+  args:
+  - -xc
+  - |
+    /tagger --token-file-path=token/token.txt --tag=((.:package-version)) --sha=((.:commit-sha)) --org=GoogleCloudPlatform --repo=((repo-name))
```

Comment:
you can't use ((.:name)) syntax in a task, it should be ((name)) and that should be set in vars for the invoking job

Reply:
make them to ((var-name)) and pass in from job var. besides. remove `sh` just use `/tagger`

Reply:
I didn't see any problem to use `load_var` in task. But we could remove `sh` and direct invoke `/tagger`

Reply:
the issue is that you now have an undeclared input to the task. it might work based on current task implementation but is not recommended. use ((name)) syntax and pass it in explicitly through vars:

yes, we can also remove /bin/sh from this command

Reply:
I didn't think the undeclared input. That's not a good implementation.